src/client/mod.rs

```diff
@@ -257,7 +257,7 @@ impl<'a, W: Write> MultipartWriter<'a, W> {
         }
 
         // always write the closing boundary, even for empty bodies
-        write!(self.inner, "--{}--", self.boundary)?;
+        write!(self.inner, "--{}--\r\n", self.boundary)?;
```

Comment:
I'll go ahead and merge this since Actix isn't moving quickly on the issue I opened. Can you add a comment here saying it's not strictly necessary but some servers expect it?